packages/nuxt-windicss/src/index.ts

```diff
@@ -58,8 +60,8 @@ export default defineNuxtModule<NuxtWindiOptions>(nuxt => ({
         'nuxt-img': 'img',
       },
     },
-  },
-  async setup(nuxtWindiOptions: NuxtWindiOptions) {
+  }),
+  async setup(nuxtWindiOptions: NuxtWindiOptions, nuxt) {
```

Comment:
`: NuxtWindiOptions` can be removed. Type is inferred from `defineNuxtModule<NuxtWindiOptions>`